developer_guides/debugability/probes/index.rst

```diff
@@ -49,8 +70,54 @@ of extraction.
 
 		echo 7,1,0 > probe_points
 
+  Please refer to host side struct sof_probe_point_desc defined in ``sound/soc/sof/probe.h``
+  or struct probe_point in ``/src/include/ipc/probe.h`` from sof for the meaning of the triplets:
+
+	.. code-block:: c
+
+		/**
+		 * Description of probe point
+		 */
+		struct probe_point {
+			uint32_t buffer_id;	/**< ID of buffer to which probe is attached */
+			uint32_t purpose;	/**< PROBE_PURPOSE_EXTRACTION or PROBE_PURPOSE_INJECTION */
+			uint32_t stream_tag;	/**< Stream tag of DMA via which data will be provided for injection.
+						 *   For extraction purposes, stream tag is ignored when received,
+						 *   but returned actual extraction stream tag via INFO function.
+						 */
+		} __attribute__((packed));
+
+  In the above example, 7 stands for the ``buffer_id`` which is a monolithic counter
+  value follows component instantiation order.
+
+  One way to find out the right instance of ``buffer_id`` is to enable dev_dbg in ``sound/sound/soc/sof/topology.c``
+  and search for widget id from the following messages:
+
+	.. code-block:: c
+
+		dev_dbg(scomp->dev, "tplg: ready widget id %d pipe %d type %d name : %s stream %s\n",
+			swidget->comp_id, index, swidget->id, tw->name,
+			strnlen(tw->sname, SNDRV_CTL_ELEM_ID_NAME_MAXLEN) > 0
+				? tw->sname : "none");
+
 - (optionally) Unpause the playback stream
 - Close the playback stream when done
 - Close the crecord tool
-- (optionally) Parse data using the probes app from sof tools; check the
-  probes app help (-h) for usage info. As a result, you will receive PCM wave files for each probe point.
+
+Data parsing
+************
+
+To construct actual waves from dumped binary, please follow the instructions at
+`Build SOF from scratch: Step 4: Build Topology and Tools <https://thesofproject.github.io/latest/getting_started/build-guide/build-from-scratch.html#step-4-build-topology-and-tools>`__ to build sof-probes, use ``-p`` for parse.
```

Comment:
So this is instruction on how to build the tool ? Not needed since the expectation is that users have completed all the getting started guide prior to this stage. You can remove, or if unsure @deb-intel can fix it up for you.